cmd/server/cadence/cadence.go

```diff
@@ -197,14 +197,14 @@ func startTestWebhookEndpoint() {
 	http.HandleFunc("/", logIncomingRequest)
 
 	fmt.Printf("Starting server for testing...\n")
-	if err := http.ListenAndServe(":8080", nil); err != nil {
+	if err := http.ListenAndServe(":8081", nil); err != nil {
```

Comment:
seems zookeeper is also using 8080

Reply:
Can we add a config for this? In case 8081 is also occupied, user can choose a different port for testing.